health/health.go

```diff
@@ -3,90 +3,134 @@ package health
 import (
 	"context"
 	"fmt"
-	"io"
-	"io/ioutil"
 	"net/http"
+	"time"
 
+	"github.com/ONSdigital/dp-api-clients-go/clientlog"
 	health "github.com/ONSdigital/dp-healthcheck/healthcheck"
 	rchttp "github.com/ONSdigital/dp-rchttp"
 	"github.com/ONSdigital/log.go/log"
 )
 
-// ErrInvalidAPIResponse is returned when an api does not respond
+var (
+	// StatusMessage contains a map of messages to service response statuses
+	StatusMessage = map[string]string{
```

Comment:
Do we want to use default messages? That would shadow the error that caused a warning or a critical, but on the other hand, this would make the error concise.

Reply:
It is not for an applications health to indicate the reason to why another app is failing; just to indicate there is an issue with an upstream application.

The only case where I think you are right to return the error in the message is if there was an issue with the client; so I have updated the health.Checker function to return this error if it occurs.